pkg/pod/entrypoint.go

```diff
@@ -335,7 +354,11 @@ func TrimSidecarPrefix(name string) string { return strings.TrimPrefix(name, sid
 // returns "step-unnamed-<step-index>" if not specified
 func StepName(name string, i int) string {
 	if name != "" {
-		return fmt.Sprintf("%s%s", stepPrefix, name)
+		return getContainerName(name)
 	}
 	return fmt.Sprintf("%sunnamed-%d", stepPrefix, i)
 }
+
+func getContainerName(name string) string {
+	return fmt.Sprintf("%s%s", stepPrefix, name)
+}
```

Comment:
this also seems like unrelated cleanup, can happen before or after this change to keep the change history clear

Reply:
We use this function repeatedly. This is something Billy called out that we should write it in a function instead of calling `fmt.Sprintf` multiple times so that it is consistent across.

Reply:
Prior to this, it was only used once so a function was not needed. Now it is to ensure consistency